src/tests/ScoreBoard.spec.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { render, cleanup } from 'react-testing-library';
+import { ScoreBoard } from '../ScoreBoard';
+
+const originalLocalStorage = global.window;
+
+const mockLocalStorage = {
+  getItem: function (key) {
+    if (key === 'userMoves') {
+      return JSON.stringify([
+        { name: 'John', moves: 20 },
+        { name: 'Andy', moves: 8 },
+        { name: 'Bill', moves: 10 },
+      ])
+    }
+  },
+  setItem: jest.fn(),
+  clear: jest.fn()
+}
+
+beforeAll(() => {
+  global.localStorage = mockLocalStorage;
+})
+
+describe('ScoreBoard component', () => {
+  it('should render three items in list if three items are stored in localStorage', () => {
+    const { container } = render(<ScoreBoard />);
+    expect(container.querySelectorAll('li')).toHaveLength(3)
```

Comment:
I think the second test is already testing that 3 items show 3 scores. Here what you should do is test that if `getItem` for the key 'userMoves' return null then nothing is rendered.